### PR TITLE
Allows specifying role attribute

### PR DESCRIPTION
**Pull request by Simonpedro**

### CODE REVIEW

File: spec/components/anchor/toast_component_spec.rb
Comment (tysongach) [inline spec/components/anchor/toast_component_spec.rb:1]: Most of the changes in this file can probably be removed…

File: spec/components/anchor/toast_component_spec.rb [hunk 1]
```diff
@@ -52,14 +52,36 @@
...
+  it 'allows specifying role attribute' do
+    component = described_class.new(role: :alert)
+
+    render_inline(component) do |c|
+      c.with_icon(icon: 'actions-check-circle-base')
+      'Checks out.'
+    end
+
+    expect(page).to have_selector('[role="alert"]')
+  end
```
Comment (tysongach) [inline spec/components/anchor/toast_component_spec.rb:75]: Looks like this is a duplicate of the spec on line +77?

**Review by tysongach — APPROVED** [review]

File: spec/components/anchor/toast_component_spec.rb
Comment (Simonpedro) [reply spec/components/anchor/toast_component_spec.rb:1]: Yes, thank you! Hadn't pushed a local change rectifying all of this

File: spec/components/anchor/toast_component_spec.rb (hunk 1 above)
Comment (Simonpedro) [reply spec/components/anchor/toast_component_spec.rb]: It was indeed, had an issue rebasing that messed it up. ty!

**Pull request merged**